index.html

```diff
@@ -1,12 +1,262 @@
 <!DOCTYPE html>
 <html lang="ru">
+
   <head>
     <meta charset="utf-8">
-    <title>HTML Academy: Седона</title>
+    <title>Гостиницы Седоны</title>
+    <link rel="stylesheet" href="styles.css">
   </head>
+
   <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
 
+    <!-- Шапка -->
+
+
+    <header class="page-header" data-test="header">
+      <nav class="page-navigation">
+        <ul class="navigation-list">
+
+          <!-- Logo -->
+          <li class="navigation-item">
+            <a class="navigation-link" href="index.html" aria-current="page">
+              <img class="page-header-logo" src="images/same/logo.svg" width="140" height="70"
+                alt="Логотип сайта города Седоны.">
+            </a>
+          </li>
+
+          <!-- Главная -->
+          <li class="navigation-item">
+            <a class="navigation-link navigation-link-current" href="page.html">Главная</a>
+          </li>
+
+          <!-- О Седоне -->
+          <li class="navigation-item">
+            <a class="navigation-link" href="about_Sedona.html">О Седоне</a>
+          </li>
+
+          <!-- Гостиницы -->
+          <li class="navigation-item">
+            <a class="navigation-link" href="hotel.html">Гостиницы</a>
+          </li>
```

Comment:
```suggestion
          <li class="navigation-item">
            <a class="navigation-link" href="catalog.html">Гостиницы</a>
          </li>
```
Ты ведь сама назвала файл с версткой гостиниц `catalog.html`. А хочешь, чтобы браузер отобразил hotel.html

Reply:
поняла свои ошибки :)